client/blocks/stats-navigation/style.scss

```diff
@@ -1,4 +1,6 @@
 .stats-navigation {
+	box-shadow: inset 0 -1px 0 #0000000d;
```

Comment:
Let's define this color as a variable and re-use it across the Stats codebase, similar to what we already do within [the horizontal bar list component](https://github.com/Automattic/wp-calypso/blob/trunk/packages/components/src/horizontal-bar-list/_stats-variables.scss).